javascript/javascript1/week3/traveltime.js

```diff
@@ -0,0 +1,15 @@
+const travelInformation = 
+{
+    speed: 50,
+    destinationDistance: 432,
+};
+  function calctime()
+  { 
+     const travelhours = (Math.floor (travelInformation.destinationDistance/travelInformation.speed));
+     const travelminutes = (travelInformation.destinationDistance%travelInformation.speed);
+     const travelTime = `Travel time is ${travelhours} hours and  ${travelminutes} minutes.`;
+     return(travelTime); 
```

Comment:
Don't add the function as a property of `travelInformation` object.
Make it like this
```js
function calctime(travelInformation) {...}

calctime(travelInformation);
```